donkeycar/parts/datastore_v2.py

```diff
@@ -233,7 +243,7 @@ def __init__(self, base_path, inputs=[], types=[], metadata=[],
         has_catalogs = False
 
         if self.manifest_path.exists():
-            self.seekeable = Seekable(self.manifest_path, read_only=self.read_only)
+            self.seekeable = Seekable(self.manifest_path, read_only=False)
```

Comment:
Is the intention here to support deletes even during read only mode ?
Please add a comment stating that.

Reply:
Yes, but we can rethink this. I see very little performance improvements by setting this, so let me remove that - actually I thought I took it out :-)